src/Upgrader/Business/Composer/CommandExecutor/UpdateCommand.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\Composer\CommandExecutor;
```

Comment:
Are you sure that it's correct location, command executor, I think command is more suitable?

Reply:
Renamed namespace